packages/yew-router/src/routable.rs

```diff
@@ -8,6 +8,11 @@ pub use yew_router_macro::Routable;
 ///
 /// Use derive macro to implement it. Although it *is* possible to implement it manually,
 /// it is discouraged.
+///
+/// # Usage
+///
+/// The functions exposed by this trait are **not** supposed to be consumed directly. Instead use
```

Comment:
It is *generally* the case that `Routable` won't be consumed directly by the user. The doc comment exists because macro introduces state and therefore care must be taken that a manual usage cleans up properly. See the above comment about side-effects.